src/pages/yonghyun/Login/Login.js

```diff
@@ -1,9 +1,90 @@
 import React from 'react';
 import { withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      canLogin: false,
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  isLogin = () => {
+    if (this.state.id.includes('@') && this.state.pw.length > 4) {
+      this.setState({
+        canLogin: true,
+      });
+    } else {
+      this.setState({
+        canLogin: false,
+      });
+    }
```

Comment:
- 다음과 같이 좀 더 간결하게 작성할 수 있습니다.
```jsx
const { id, pw } = this.state;
this.setState({
  canLogin: id.includes('@') && pw.length > 4
})
```
- 사실 canLogin은 state로 관리할 필요가 없는 정보이기 때문에 state에서 제거하면 필요없는 메서드일 것 같네요. 참고만 해주세요!

Reply:
아 state에서 따로 지정해줄 필요가 없이 바로 계산해줘도 되는 부분이었네요 
생각치도 못했던 부분인데 감사합니다~!
```suggestion
   const { id, pw } = this.state;
   this.setState({
          canLogin: id.includes('@') && pw.length > 4
    })
```